### DISCUSSION TITLE
where is located the defaulft conf for Log Retention?

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#5463] [Unsupported Versions] [answered]
Dear all,

Please can someone help me with the log retention of Security Onion 2.3.70. I'm trying to manually configure the  Log Retention for the services (Suricata, Wazuh, elasticsearch). Please can you guide me to the default configuration file?

Thanks in advance.

**Comment by dougburks** [comment, marked as answer]
Take a look at the Curator section of the documentation:
https://docs.securityonion.net/en/2.3/curator.html

**Reply by ITgeevoSOC** [reply]
Thanks for your help

**Comment by ben-sec** [comment]
cur_close_days is set to null in my manager minion config file, instead there are plenty of index_settings in the global.sls like

```
index_settings:
      so-beats:
        shards: 1
        warm: 7
        close: 30
        delete: 365
...
```

I was not able to find a documentation of these settings, but my understanding ist, that ES logs get deleted after 365 days. Correct?

Additionally log_size_limit is set to 474. Is this MB, GB, TB, ...? Thanks in advance!

Cheers, Ben

**Reply by dougburks** [reply]
@ben-sec From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.
